# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../udev/overwrite_rules.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-10-11 18:27:00.000000000 -0400
+++ ./Makefile	2007-10-11 18:29:46.000000000 -0400
@@ -227,13 +227,9 @@
 
 install-config:
 	$(INSTALL) -d $(DESTDIR)$(configdir)/rules.d
-	@ if [ ! -r $(DESTDIR)$(configdir)/udev.conf ]; then \
-		$(INSTALL_DATA) etc/udev/udev.conf $(DESTDIR)$(configdir); \
-	fi
+	@ $(INSTALL_DATA) etc/udev/udev.conf $(DESTDIR)$(configdir);
 	@ for i in etc/udev/rules.d/*.rules; do \
-		if [ ! -r $(DESTDIR)$(configdir)/rules.d/$$(basename $$i) ]; then \
 			$(INSTALL_DATA) $$i $(DESTDIR)$(configdir)/rules.d; \
-		fi \
 	done
 	@ extras="$(EXTRAS)"; for target in $$extras; do \
 		$(MAKE) -C $$target $@ || exit 1; \
